static/app/components/slider/index.tsx

```diff
@@ -0,0 +1,382 @@
+    forwardedRef,
+    () => {
+      if (nThumbs > 1) {
+        return refs.current;
+      }
+      return refs.current[0];
+    },
+    [nThumbs]
+  );
+
+  const getFormattedValue = useCallback(
+    (val: number) => {
+      // Special formatting when `allowedValues` is specified, in which case `val` acts
+      // like an index for `allowedValues`.
+      if (allowedValues) {
+        return formatLabel
+          ? formatLabel(allowedValues[val])
+          : state.getFormattedValue(allowedValues[val]);
+      }
+
+      return formatLabel ? formatLabel(val) : state.getFormattedValue(val);
+    },
+    [formatLabel, state, allowedValues]
+  );
+
+  const selectedRange =
+    nThumbs > 1
+      ? [Math.min(...state.values), Math.max(...state.values)]
+      : [min, state.values[0]];
+
+  return (
+    <SliderGroup {...groupProps} className={className}>
+      {label && (
+        <SliderLabelWrapper className="label-container">
+          <SliderLabel {...labelProps}>{label}</SliderLabel>
+          <SliderLabelOutput {...outputProps}>
+            {nThumbs > 1
+              ? `${getFormattedValue(selectedRange[0])}–${getFormattedValue(
+                  selectedRange[1]
+                )}`
+              : getFormattedValue(selectedRange[1])}
+          </SliderLabelOutput>
+        </SliderLabelWrapper>
+      )}
+
+      <SliderTrack
+        ref={trackRef}
+        {...trackProps}
+        hasThumbLabels={showThumbLabels && !label}
+        hasTickLabels={showTickLabels && allTickValues.length > 0}
+      >
+        <SliderLowerTrack
+          role="presentation"
+          disabled={disabled}
+          error={error}
+          style={{
+            left: `${state.getValuePercent(selectedRange[0]) * 100}%`,
+            right: `${100 - state.getValuePercent(selectedRange[1]) * 100}%`,
+          }}
+        />
+
+        {allTickValues.map((tickValue, index) => (
+          <SliderTick
+            key={tickValue}
+            aria-hidden
+            error={error}
+            disabled={disabled}
+            inSelection={tickValue >= selectedRange[0] && tickValue <= selectedRange[1]}
+            style={{left: `${(state.getValuePercent(tickValue) * 100).toFixed(2)}%`}}
+            justifyContent={
+              index === 0
+                ? 'start'
+                : index === allTickValues.length - 1
+                ? 'end'
+                : 'center'
+            }
+          >
+            {showTickLabels && (
+              <SliderTickLabel>{getFormattedValue(tickValue)}</SliderTickLabel>
+            )}
+          </SliderTick>
+        ))}
+
+        {[...new Array(nThumbs)].map((_, index) => (
+          <SliderThumb
+            ref={node => {
+              if (!node) {
+                return;
+              }
+
+              refs.current = [
+                ...refs.current.slice(0, index),
+                node,
+                ...refs.current.slice(index + 1),
+              ];
+            }}
```

Comment:
Build a list of refs to each thumb's input element. 

**If there is only one thumb**, then `Slider`'s ref will point to the input element for just that thumb
<img width="619" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/46090b07-737b-49fc-8d30-bffde1c9e98d">
```js 
{current: <input … />})
```

**If there are two or more thumbs**, then `Slider`'s ref will point to an array of input elements, each corresponding to a thumb.
<img width="619" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/5396a127-5f21-4e7b-bd92-3e3488317ce0">
```js 
{current: [<input … />, <input … />]})
```